Postgresql: error with schema name with upper letter

Description: 

2.0.0-beta5:
1.  schema name: sch_testa
2.  creation of siard works:
      Obtaining table structure for sch_testa.users
      Database structure converted
3. schema renamed: sch_testA
4. creation of siard error:
      Obtaining table structure for sch_testA.users
      ERROR Connection error while importing/exporting
      ERROR Fatal error while converting the database (SQL error while connecting)

Attach the dbptk-app.log.txt file below.
2016-08-02 14:30:06,430 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `sch_testA.users.name` (format: schema.table.column) has original type `VARCHAR` and was converted to the standard type `CHARACTER VARYING(2147483647)`; and type: Type{description='null', originalTypeName='varchar', sql99TypeName='CHARACTER VARYING(2147483647)', sql2008TypeName='CHARACTER VARYING(2147483647)'}
2016-08-02 14:30:06,430 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 12
Type name: varchar
Column size: 2147483647
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 2
Is Nullable: true
Is auto increment: false
Calculated type: SimpleTypeString
ColumnType hash: -547542414

2016-08-02 14:30:06,460 [main] DEBUG (c.d.m.j.i.JDBCImportModule) count query: SELECT COUNT(*) FROM "sch_testa"."users"
2016-08-02 14:30:06,490 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Closing connection to source database
2016-08-02 14:30:06,490 [main] ERROR (c.d.Main) Connection error while importing/exporting
com.databasepreservation.model.exception.ModuleException: SQL error while connecting
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1565)
Caused by: org.postgresql.util.PSQLException: ERROR: relation "sch_testa.users" does not exist
  Position: 22

No:
d:\projekti\E-ARK>java -Xmx1000m -Dfile.encoding=UTF-8 -jar dbptk-app-2.0.0-SNAPSHOT-246-1.jar -i postgresql -ih pluton -idb testA -iu xxx -ip xxx -ide -ipn 5432 -e siard-2 -ef out_pg1.siard
      Converting database: postgresql to siard-2
      Initializing target database
      Converting database structure
      Obtaining table structure for public.arhiv_log
      Obtaining table structure for public.tabela1
      Obtaining table structure for sch_testA.users
      Database structure converted
      Getting contents from table 'public.arhiv_log'
      Total of 11197 row(s) processed
      Obtained contents from table 'public.arhiv_log'
      Getting contents from table 'public.tabela1'
      Total of 2239 row(s) processed
      Obtained contents from table 'public.tabela1'
      Getting contents from table 'sch_testA.users'
ERROR Connection error while importing/exporting
ERROR Fatal error while converting the database (SQL error while connecting)
      Disabling connection encryption (for modules that support it) and trying again.
      Converting database: postgresql to siard-2
      Initializing target database
      Converting database structure
      Obtaining table structure for public.arhiv_log
      Obtaining table structure for public.tabela1
      Obtaining table structure for sch_testA.users
      Database structure converted
      Getting contents from table 'public.arhiv_log'
      Total of 11197 row(s) processed
      Obtained contents from table 'public.arhiv_log'
      Getting contents from table 'public.tabela1'
      Total of 2239 row(s) processed
      Obtained contents from table 'public.tabela1'
      Getting contents from table 'sch_testA.users'
ERROR Connection error while importing/exporting
ERROR Fatal error while converting the database (SQL error while connecting)
      A report was generated with a listing of information that was modified during the conversion.
      The report file is located at d:\projekti\E-ARK\dbptk-report-20160811140733748.txt
      Please report any problems at https://github.com/keeps/db-preservation-toolkit/issues/new

2016-08-11 14:07:35,708 [main] DEBUG (c.d.m.j.i.JDBCImportModule) count query: SELECT COUNT(*) FROM "sch_testA"."users"
2016-08-11 14:07:35,710 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Counted 2 rows
2016-08-11 14:07:35,721 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Database structure obtained
2016-08-11 14:07:35,721 [main] INFO  (c.d.m.j.i.JDBCImportModule) Database structure converted
2016-08-11 14:07:35,722 [main] INFO  (c.d.m.j.i.JDBCImportModule) Getting contents from table 'public.arhiv_log'
2016-08-11 14:07:35,723 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) query: SELECT tabela, polje, vrednost, vrsta, sprememba, uporabnik, datum FROM public.arhiv_log
2016-08-11 14:07:36,471 [main] INFO  (c.d.m.j.i.JDBCImportModule) Total of 11197 row(s) processed
2016-08-11 14:07:36,472 [main] INFO  (c.d.m.j.i.JDBCImportModule) Obtained contents from table 'public.arhiv_log'
2016-08-11 14:07:36,472 [main] INFO  (c.d.m.j.i.JDBCImportModule) Getting contents from table 'public.tabela1'
2016-08-11 14:07:36,472 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) query: SELECT num1, num2, str1, bool1 FROM public.tabela1
2016-08-11 14:07:36,532 [main] INFO  (c.d.m.j.i.JDBCImportModule) Total of 2239 row(s) processed
2016-08-11 14:07:36,533 [main] INFO  (c.d.m.j.i.JDBCImportModule) Obtained contents from table 'public.tabela1'
2016-08-11 14:07:36,533 [main] INFO  (c.d.m.j.i.JDBCImportModule) Getting contents from table 'sch_testA.users'
2016-08-11 14:07:36,533 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) query: SELECT id, name FROM sch_testA.users
2016-08-11 14:07:36,535 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Closing connection to source database
2016-08-11 14:07:36,535 [main] ERROR (c.d.Main) Connection error while importing/exporting
com.databasepreservation.model.exception.ModuleException: SQL error while connecting
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1565)
	at com.databasepreservation.Main.run(Main.java:126)
	at com.databasepreservation.Main.internal_main(Main.java:84)
	at com.databasepreservation.Main.main(Main.java:62)
Caused by: org.postgresql.util.PSQLException: ERROR: relation "sch_testa.users" does not exist
  Position: 22
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2284)

at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2284)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2003)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:200)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:424)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:321)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:284)
	at com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCImportModule.getTableRawData(PostgreSQLJDBCImportModule.java:168)
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1538)
	... 3 common frames omitted
2016-08-11 14:07:36,535 [main] ERROR (c.d.Main) Fatal error while converting the database (SQL error while connecting)
com.databasepreservation.model.exception.ModuleException: SQL error while connecting
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1565)
	at com.databasepreservation.Main.run(Main.java:126)
	at com.databasepreservation.Main.internal_main(Main.java:84)
	at com.databasepreservation.Main.main(Main.java:62)
Caused by: org.postgresql.util.PSQLException: ERROR: relation "sch_testa.users" does not exist
  Position: 22
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2284)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2003)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:200)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:424)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:321)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:284)
	at com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCImportModule.getTableRawData(PostgreSQLJDBCImportModule.java:168)
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1538)
	... 3 common frames omitted
2016-08-11 14:07:36,536 [main] INFO  (c.d.m.Reporter) A report was generated with a listing of information that was modified during the conversion.
2016-08-11 14:07:36,537 [main] INFO  (c.d.m.Reporter) The report file is located at d:\projekti\E-ARK\dbptk-report-20160811140733748.txt
2016-08-11 14:07:36,537 [main] INFO  (c.d.Main) Please report any problems at https://github.com/keeps/db-preservation-toolkit/issues/new
2016-08-11 14:07:36,537 [main] DEBUG (c.d.Main) #########################################################
2016-08-11 14:07:36,537 [main] DEBUG (c.d.Main) #   FINISH-ID-7597e74a-e2c3-41d4-8d7a-ca571b18877a
2016-08-11 14:07:36,537 [main] DEBUG (c.d.Main) #   FINISH v2.0.0-SNAPSHOT-246-1
2016-08-11 14:07:36,537 [main] DEBUG (c.d.Main) #   EXIT CODE: 4
2016-08-11 14:07:36,537 [main] DEBUG (c.d.Main) #########################################################

two (same?) errors, but maybe we will need to continue on Wednesday: 
2016-08-11 15:44:22,685 [main] DEBUG (c.d.m.j.i.JDBCImportModule) count query: SELECT COUNT(*) FROM "sch_testA"."users"
2016-08-11 15:44:22,686 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Counted 2 rows
2016-08-11 15:44:22,700 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Database structure obtained
2016-08-11 15:44:22,701 [main] INFO  (c.d.m.j.i.JDBCImportModule) Database structure converted
2016-08-11 15:44:22,705 [main] INFO  (c.d.m.j.i.JDBCImportModule) Getting contents from table 'public.arhiv_log'
2016-08-11 15:44:22,736 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) query: SELECT tabela, polje, vrednost, vrsta, sprememba, uporabnik, datum FROM public.arhiv_log
2016-08-11 15:44:23,585 [main] INFO  (c.d.m.j.i.JDBCImportModule) Total of 11197 row(s) processed
2016-08-11 15:44:23,589 [main] INFO  (c.d.m.j.i.JDBCImportModule) Obtained contents from table 'public.arhiv_log'
2016-08-11 15:44:23,591 [main] INFO  (c.d.m.j.i.JDBCImportModule) Getting contents from table 'public.tabela1'
2016-08-11 15:44:23,596 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) query: SELECT num1, num2, str1, bool1 FROM public.tabela1
2016-08-11 15:44:23,660 [main] INFO  (c.d.m.j.i.JDBCImportModule) Total of 2239 row(s) processed
2016-08-11 15:44:23,663 [main] INFO  (c.d.m.j.i.JDBCImportModule) Obtained contents from table 'public.tabela1'
2016-08-11 15:44:23,665 [main] INFO  (c.d.m.j.i.JDBCImportModule) Getting contents from table 'sch_testA.users'
2016-08-11 15:44:23,669 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) query: SELECT id, name FROM sch_testA.users
2016-08-11 15:44:23,672 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Closing connection to source database
2016-08-11 15:44:23,674 [main] ERROR (c.d.Main) Connection error while importing/exporting
com.databasepreservation.model.exception.ModuleException: SQL error while connecting
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1565)
	at com.databasepreservation.Main.run(Main.java:126)
	at com.databasepreservation.Main.internal_main(Main.java:80)
	at com.databasepreservation.Main.main(Main.java:62)
Caused by: org.postgresql.util.PSQLException: ERROR: relation "sch_testa.users" does not exist
  Position: 22
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2284)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2003)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:200)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:424)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:321)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:284)
	at com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCImportModule.getTableRawData(PostgreSQLJDBCImportModule.java:168)
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1538)
	... 3 common frames omitted
2016-08-11 15:44:23,676 [main] ERROR (c.d.Main) Fatal error while converting the database (SQL error while connecting)
com.databasepreservation.model.exception.ModuleException: SQL error while connecting
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1565)
	at com.databasepreservation.Main.run(Main.java:126)
	at com.databasepreservation.Main.internal_main(Main.java:80)
	at com.databasepreservation.Main.main(Main.java:62)
Caused by: org.postgresql.util.PSQLException: ERROR: relation "sch_testa.users" does not exist
  Position: 22
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2284)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2003)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:200)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:424)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:321)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:284)
	at com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCImportModule.getTableRawData(PostgreSQLJDBCImportModule.java:168)
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1538)
	... 3 common frames omitted
2016-08-11 15:44:23,679 [main] INFO  (c.d.Main) Disabling connection encryption (for modules that support it) and trying again.
2016-08-11 15:44:23,683 [main] DEBUG (c.d.m.Reporter) moduleParameters, module: postgresql with parameters **Import module:** postgresql
- hostname = pluton
- database = testA
- username = pg_boris
- password = <filtered>
- port number = 5432
2016-08-11 15:44:23,684 [main] DEBUG (c.d.m.Reporter) moduleParameters, module: siard-2 with parameters **Export module:** siard-2
- file = d:\projekti\E-ARK\out_pg1.siard
- compress = false
- pretty xml = false
2016-08-11 15:44:23,684 [main] INFO  (c.d.Main) Converting database: postgresql to siard-2
2016-08-11 15:44:23,686 [main] INFO  (c.d.m.j.i.JDBCImportModule) Initializing target database
2016-08-11 15:44:23,693 [main] INFO  (c.d.m.j.i.JDBCImportModule) Converting database structure
2016-08-11 15:44:23,695 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) Loading JDBC Driver org.postgresql.Driver
2016-08-11 15:44:23,696 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) Getting connection
2016-08-11 15:44:23,705 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) Connected
2016-08-11 15:44:23,705 [main] DEBUG (c.d.m.j.i.JDBCImportModule) driver version: PostgreSQL 9.4.1208.jre7
2016-08-11 15:44:23,712 [main] INFO  (c.d.m.j.i.JDBCImportModule) Obtaining table structure for public.arhiv_log
2016-08-11 15:44:23,719 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `public.arhiv_log.tabela` (format: schema.table.column) has original type `VARCHAR` and was converted to the standard type `CHARACTER VARYING(2147483647)`; and type: Type{description='null', originalTypeName='varchar', sql99TypeName='CHARACTER VARYING(2147483647)', sql2008TypeName='CHARACTER VARYING(2147483647)'}
2016-08-11 15:44:23,719 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 12
Type name: varchar
Column size: 2147483647
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 1
Is Nullable: false
Is auto increment: false
Calculated type: SimpleTypeString
ColumnType hash: -547542414

2016-08-11 15:44:23,719 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `public.arhiv_log.polje` (format: schema.table.column) has original type `VARCHAR` and was converted to the standard type `CHARACTER VARYING(2147483647)`; and type: Type{description='null', originalTypeName='varchar', sql99TypeName='CHARACTER VARYING(2147483647)', sql2008TypeName='CHARACTER VARYING(2147483647)'}
2016-08-11 15:44:23,719 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 12
Type name: varchar
Column size: 2147483647
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 2
Is Nullable: false
Is auto increment: false
Calculated type: SimpleTypeString
ColumnType hash: -547542414

2016-08-11 15:44:23,719 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `public.arhiv_log.vrednost` (format: schema.table.column) has original type `VARCHAR` and was converted to the standard type `CHARACTER VARYING(2147483647)`; and type: Type{description='null', originalTypeName='varchar', sql99TypeName='CHARACTER VARYING(2147483647)', sql2008TypeName='CHARACTER VARYING(2147483647)'}
2016-08-11 15:44:23,719 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 12
Type name: varchar
Column size: 2147483647
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 3
Is Nullable: true
Is auto increment: false
Calculated type: SimpleTypeString
ColumnType hash: -547542414

2016-08-11 15:44:23,720 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `public.arhiv_log.vrsta` (format: schema.table.column) has original type `CHARACTER(1)` and was converted to the standard type `CHARACTER(1)`; and type: Type{description='null', originalTypeName='CHARACTER(1)', sql99TypeName='CHARACTER(1)', sql2008TypeName='CHARACTER(1)'}
2016-08-11 15:44:23,720 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 1
Type name: bpchar
Column size: 1
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 4
Is Nullable: false
Is auto increment: false
Calculated type: SimpleTypeString
ColumnType hash: 314841966

2016-08-11 15:44:23,720 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `public.arhiv_log.sprememba` (format: schema.table.column) has original type `VARCHAR` and was converted to the standard type `CHARACTER VARYING(2147483647)`; and type: Type{description='null', originalTypeName='varchar', sql99TypeName='CHARACTER VARYING(2147483647)', sql2008TypeName='CHARACTER VARYING(2147483647)'}
2016-08-11 15:44:23,720 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 12
Type name: varchar
Column size: 2147483647
Decimal digits: 0
Radix: 10
Remarks: This is a comment
Default value: null
Index: 5
Is Nullable: false
Is auto increment: false
Calculated type: SimpleTypeString
ColumnType hash: -547542414

2016-08-11 15:44:23,720 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `public.arhiv_log.uporabnik` (format: schema.table.column) has original type `VARCHAR` and was converted to the standard type `CHARACTER VARYING(2147483647)`; and type: Type{description='null', originalTypeName='varchar', sql99TypeName='CHARACTER VARYING(2147483647)', sql2008TypeName='CHARACTER VARYING(2147483647)'}
2016-08-11 15:44:23,720 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 12
Type name: varchar
Column size: 2147483647
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 6
Is Nullable: true
Is auto increment: false
Calculated type: SimpleTypeString
ColumnType hash: -547542414

2016-08-11 15:44:23,720 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `public.arhiv_log.datum` (format: schema.table.column) has original type `TIMESTAMP` and was converted to the standard type `TIMESTAMP`; and type: Type{description='null', originalTypeName='timestamp', sql99TypeName='TIMESTAMP', sql2008TypeName='TIMESTAMP'}
2016-08-11 15:44:23,720 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 93
Type name: timestamp
Column size: 29
Decimal digits: 6
Radix: 10
Remarks: null
Default value: null
Index: 7
Is Nullable: true
Is auto increment: false
Calculated type: SimpleTypeDateTime
ColumnType hash: 1852114857

2016-08-11 15:44:23,751 [main] DEBUG (c.d.m.j.i.JDBCImportModule) count query: SELECT COUNT(*) FROM "public"."arhiv_log"
2016-08-11 15:44:23,754 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Counted 11197 rows
2016-08-11 15:44:23,755 [main] INFO  (c.d.m.j.i.JDBCImportModule) Obtaining table structure for public.tabela1
2016-08-11 15:44:23,760 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `public.tabela1.num1` (format: schema.table.column) has original type `NUMERIC` and was converted to the standard type `NUMERIC(131089)`; and type: Type{description='null', originalTypeName='numeric', sql99TypeName='NUMERIC(131089)', sql2008TypeName='NUMERIC(131089)'}
2016-08-11 15:44:23,761 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 2
Type name: numeric
Column size: 131089
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 1
Is Nullable: true
Is auto increment: false
Calculated type: SimpleTypeNumericExact
ColumnType hash: 1596766801

2016-08-11 15:44:23,761 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `public.tabela1.num2` (format: schema.table.column) has original type `NUMERIC` and was converted to the standard type `NUMERIC(131089)`; and type: Type{description='null', originalTypeName='numeric', sql99TypeName='NUMERIC(131089)', sql2008TypeName='NUMERIC(131089)'}
2016-08-11 15:44:23,761 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 2
Type name: numeric
Column size: 131089
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 2
Is Nullable: true
Is auto increment: false
Calculated type: SimpleTypeNumericExact
ColumnType hash: 1596766801

2016-08-11 15:44:23,762 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `public.tabela1.str1` (format: schema.table.column) has original type `VARCHAR` and was converted to the standard type `CHARACTER VARYING(2147483647)`; and type: Type{description='null', originalTypeName='varchar', sql99TypeName='CHARACTER VARYING(2147483647)', sql2008TypeName='CHARACTER VARYING(2147483647)'}
2016-08-11 15:44:23,762 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 12
Type name: varchar
Column size: 2147483647
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 3
Is Nullable: true
Is auto increment: false
Calculated type: SimpleTypeString
ColumnType hash: -547542414

2016-08-11 15:44:23,762 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `public.tabela1.bool1` (format: schema.table.column) has original type `BOOL` and was converted to the standard type `BOOLEAN`; and type: Type{description='null', originalTypeName='bool', sql99TypeName='BOOLEAN', sql2008TypeName='BOOLEAN'}
2016-08-11 15:44:23,762 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: -7
Type name: bool
Column size: 1
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 4
Is Nullable: true
Is auto increment: false
Calculated type: SimpleTypeBoolean
ColumnType hash: -1444026573

2016-08-11 15:44:23,786 [main] DEBUG (c.d.m.j.i.JDBCImportModule) count query: SELECT COUNT(*) FROM "public"."tabela1"
2016-08-11 15:44:23,787 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Counted 2239 rows
2016-08-11 15:44:23,793 [main] INFO  (c.d.m.j.i.JDBCImportModule) Obtaining table structure for sch_testA.users
2016-08-11 15:44:23,799 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `sch_testA.users.id` (format: schema.table.column) has original type `INT4` and was converted to the standard type `INTEGER`; and type: Type{description='null', originalTypeName='int4', sql99TypeName='INTEGER', sql2008TypeName='INTEGER'}
2016-08-11 15:44:23,799 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 4
Type name: int4
Column size: 10
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 1
Is Nullable: false
Is auto increment: false
Calculated type: SimpleTypeNumericExact
ColumnType hash: -1058864868

2016-08-11 15:44:23,800 [main] DEBUG (c.d.m.Reporter) dataTypeChangedOnImport, invoker: com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCDatatypeImporter; message: - Type conversion in import module: in `sch_testA.users.name` (format: schema.table.column) has original type `VARCHAR` and was converted to the standard type `CHARACTER VARYING(2147483647)`; and type: Type{description='null', originalTypeName='varchar', sql99TypeName='CHARACTER VARYING(2147483647)', sql2008TypeName='CHARACTER VARYING(2147483647)'}
2016-08-11 15:44:23,800 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Data type: 12
Type name: varchar
Column size: 2147483647
Decimal digits: 0
Radix: 10
Remarks: null
Default value: null
Index: 2
Is Nullable: true
Is auto increment: false
Calculated type: SimpleTypeString
ColumnType hash: -547542414

2016-08-11 15:44:23,822 [main] DEBUG (c.d.m.j.i.JDBCImportModule) count query: SELECT COUNT(*) FROM "sch_testA"."users"
2016-08-11 15:44:23,823 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Counted 2 rows
2016-08-11 15:44:23,834 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Database structure obtained
2016-08-11 15:44:23,834 [main] INFO  (c.d.m.j.i.JDBCImportModule) Database structure converted
2016-08-11 15:44:23,836 [main] INFO  (c.d.m.j.i.JDBCImportModule) Getting contents from table 'public.arhiv_log'
2016-08-11 15:44:23,840 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) query: SELECT tabela, polje, vrednost, vrsta, sprememba, uporabnik, datum FROM public.arhiv_log
2016-08-11 15:44:24,587 [main] INFO  (c.d.m.j.i.JDBCImportModule) Total of 11197 row(s) processed
2016-08-11 15:44:24,591 [main] INFO  (c.d.m.j.i.JDBCImportModule) Obtained contents from table 'public.arhiv_log'
2016-08-11 15:44:24,594 [main] INFO  (c.d.m.j.i.JDBCImportModule) Getting contents from table 'public.tabela1'
2016-08-11 15:44:24,597 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) query: SELECT num1, num2, str1, bool1 FROM public.tabela1
2016-08-11 15:44:24,656 [main] INFO  (c.d.m.j.i.JDBCImportModule) Total of 2239 row(s) processed
2016-08-11 15:44:24,659 [main] INFO  (c.d.m.j.i.JDBCImportModule) Obtained contents from table 'public.tabela1'
2016-08-11 15:44:24,661 [main] INFO  (c.d.m.j.i.JDBCImportModule) Getting contents from table 'sch_testA.users'
2016-08-11 15:44:24,666 [main] DEBUG (c.d.m.p.i.PostgreSQLJDBCImportModule) query: SELECT id, name FROM sch_testA.users
2016-08-11 15:44:24,667 [main] DEBUG (c.d.m.j.i.JDBCImportModule) Closing connection to source database
2016-08-11 15:44:24,668 [main] ERROR (c.d.Main) Connection error while importing/exporting
com.databasepreservation.model.exception.ModuleException: SQL error while connecting
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1565)
	at com.databasepreservation.Main.run(Main.java:126)
	at com.databasepreservation.Main.internal_main(Main.java:84)
	at com.databasepreservation.Main.main(Main.java:62)
Caused by: org.postgresql.util.PSQLException: ERROR: relation "sch_testa.users" does not exist
  Position: 22
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2284)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2003)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:200)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:424)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:321)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:284)
	at com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCImportModule.getTableRawData(PostgreSQLJDBCImportModule.java:168)
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1538)
	... 3 common frames omitted
2016-08-11 15:44:24,671 [main] ERROR (c.d.Main) Fatal error while converting the database (SQL error while connecting)
com.databasepreservation.model.exception.ModuleException: SQL error while connecting
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1565)
	at com.databasepreservation.Main.run(Main.java:126)
	at com.databasepreservation.Main.internal_main(Main.java:84)
	at com.databasepreservation.Main.main(Main.java:62)
Caused by: org.postgresql.util.PSQLException: ERROR: relation "sch_testa.users" does not exist
  Position: 22
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2284)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2003)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:200)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:424)
	at org.postgresql.jdbc.PgStatement.executeWithFlags(PgStatement.java:321)
	at org.postgresql.jdbc.PgStatement.executeQuery(PgStatement.java:284)
	at com.databasepreservation.modules.postgreSql.in.PostgreSQLJDBCImportModule.getTableRawData(PostgreSQLJDBCImportModule.java:168)
	at com.databasepreservation.modules.jdbc.in.JDBCImportModule.getDatabase(JDBCImportModule.java:1538)
	... 3 common frames omitted
2016-08-11 15:44:24,676 [main] INFO  (c.d.m.Reporter) A report was generated with a listing of information that was modified during the conversion.
2016-08-11 15:44:24,678 [main] INFO  (c.d.m.Reporter) The report file is located at d:\projekti\E-ARK\dbptk-report-20160811154422386.txt
2016-08-11 15:44:24,681 [main] INFO  (c.d.Main) Please report any problems at https://github.com/keeps/db-preservation-toolkit/issues/new
2016-08-11 15:44:24,683 [main] DEBUG (c.d.Main) #########################################################
2016-08-11 15:44:24,684 [main] DEBUG (c.d.Main) #   FINISH-ID-c38e887d-0936-40d2-9172-0ffeb71a3c4c
2016-08-11 15:44:24,684 [main] DEBUG (c.d.Main) #   FINISH v2.0.0-SNAPSHOT-246-2
2016-08-11 15:44:24,684 [main] DEBUG (c.d.Main) #   EXIT CODE: 4
2016-08-11 15:44:24,684 [main] DEBUG (c.d.Main) #########################################################

--- a/dbptk-core/src/main/java/com/databasepreservation/modules/postgreSql/PostgreSQLHelper.java
+++ b/dbptk-core/src/main/java/com/databasepreservation/modules/postgreSql/PostgreSQLHelper.java
@@ -46,13 +46,11 @@ public class PostgreSQLHelper extends SQLHelper {
 
   @Override
   public String escapeSchemaName(String schema) {
-    schema = schema.toLowerCase();
     return getStartQuote() + schema + getEndQuote();
   }
 
   @Override
   public String escapeTableName(String table) {
-    table = table.toLowerCase();
     return getStartQuote() + table + getEndQuote();
   }
 

--- a/dbptk-core/src/main/java/com/databasepreservation/modules/postgreSql/in/PostgreSQLJDBCImportModule.java
+++ b/dbptk-core/src/main/java/com/databasepreservation/modules/postgreSql/in/PostgreSQLJDBCImportModule.java
@@ -148,7 +148,7 @@ public class PostgreSQLJDBCImportModule extends JDBCImportModule {
       if (column.getType() instanceof ComposedTypeStructure) {
         udtColumns.add(column);
       }
-      columnNames.add(column.getName().toLowerCase());
+      columnNames.add(column.getName());
       query.append(separator).append(column.getName());
       separator = ", ";
     }

